### PR TITLE
Update readme with global install command

### PR DESCRIPTION
**Pull request by cgmartin**
Also adds some NPM badges to the README (dependencies and version).

A new publish isn't required, but it would update NPM's readme page.

### CODE REVIEW

**Comment by cgmartin** [comment]
Just saw your note about not recommended to install it globally. We can close this if need be.

**Comment by johnpapa** [comment]
I like the changes, but just add a note that we recommend local. K?

**Comment by cgmartin** [comment]
What do you think of these new changes? (Clarified local vs global install and usage)

File: README.md [hunk 1]
```diff
@@ -10,15 +13,44 @@ When creating a SPA there are routes that are only known to the browser. For exa
...
-The default behavior serves from the current folder, opens a browser, and applies a HTML5 route fallback to `./index.html`.
+The recommended installation method is a local NPM install for your project:
+```bash
+$ npm install lite-server --save-dev
+```
 
+...and add a "script" entry within your project's `package.json` file:
+```
+# package.json
+...
```
Comment (johnpapa) [inline README.md]: i think you want ticks here

**Comment by cgmartin** [comment]
Fixed

**Pull request merged**